src/main/java/com/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -1,4 +1,69 @@
 package com.novi.TechItEasy.controllers;
 
+import com.novi.TechItEasy.exceptions.RecordNotFoundException;
+import com.novi.TechItEasy.model.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/techItEasy")
 public class TelevisionController {
+    private List<Television> tv;// 1. declare the list
+
+    public TelevisionController() {
+        tv = new ArrayList<>();// 2. initialise the list
+        Television tele = new Television(); // instantiate the class Television
+        tele.setBrandName("onida");//setting the attributes to the instance of television
+        tele.setTeleType("LCD");
+        tv.add(tele); // 3. adding list to instance of television class
+    }
+
+    @GetMapping(value = "/televisions")
```

Comment:
Je hebt bovenaan de klasse al `@RequestMapping("/techiteasy")` staan. Het voordeel daarvan is dat je niet het path van elke mapping hoeft te beginnen met "/techiteasy", dat wordt automatisch gedaan. Nu begin je echter welk elke mapping met "/televisions" wat het nut van de `@requestMapping` annotatie te niet doet. 
Je kunt wel de bovneste annotatie aanpassen naar `@Requestmapping("/techiteasy/televisions")` en die "/televisions" in de get/put/post/delete-mappings weghalen.